### PR TITLE
[integration_test] Fix link to integration test for web section in `README.md`

### PR DESCRIPTION
**Pull request by nilsreichardt**
The link for `download and run` pointed to a non-existing headline. Therefore, I fixed the link.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: packages/integration_test/README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ flutter drive \
 ### Web
 
 Make sure you have [enabled web support](https://flutter.dev/docs/get-started/web#set-up)
-then [download and run](https://flutter.dev/docs/cookbook/testing/integration/introduction#6b-web)
+then [download and run](https://docs.flutter.dev/cookbook/testing/integration/introduction#5b-web)
```
Comment (nilsreichardt) [inline packages/integration_test/README.md:86]:
```suggestion
then [download and run](https://flutter.dev/docs/cookbook/testing/integration/introduction#5b-web)
```

Comment (Hixie) [reply packages/integration_test/README.md]: Actually I think `docs.flutter.dev` is correct, you were right the first time.

File: packages/integration_test/README.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ flutter drive \
 ### Web
 
 Make sure you have [enabled web support](https://flutter.dev/docs/get-started/web#set-up)
-then [download and run](https://flutter.dev/docs/cookbook/testing/integration/introduction#6b-web)
+then [download and run](https://flutter.dev/docs/cookbook/testing/integration/introduction#5b-web)
```
Comment (nilsreichardt) [inline packages/integration_test/README.md:86]:
```suggestion
then [download and run](https://docs.flutter.dev/cookbook/testing/integration/introduction#5b-web)
```

File: packages/integration_test/README.md (hunk 1 above)
Comment (nilsreichardt) [reply packages/integration_test/README.md:86]: Oh, thanks ❤️ I fixed it 👍

**Comment by nilsreichardt** [comment]
@Hixie Can you assign someone to review this?

**Review by Hixie — APPROVED** [review]
LGTM

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Comment by Hixie** [comment]
Thanks @nilsreichardt!

**Pull request merged**